.travis.yml

```diff
@@ -2,18 +2,19 @@ sudo: false
 language: php
 
 php:
-    - 5.4
-    - 5.5
     - 5.6
     - 7.0
+    - 7.1
+    - 7.2
+    - 7.3
+    - 7.4
     - hhvm
 
 matrix:
     allow_failures:
         - php: hhvm
 
 before_script:
-    - composer self-update
     - composer install --prefer-dist --no-interaction
 
-script: phpunit --coverage-text
+script: ./vendor/bin/phpunit --coverage-text
```

Comment:
I noticed that no matter what version of PHPUnit was installed via composer, travis was always running the version that was pre-installed on the host, which was causing PHP 7.0 and 7.1 to fail (the version installed on travis hosts for those two php versions actually isn't compatible with them).